pkg/api/v1/register.go

```diff
@@ -86,6 +86,10 @@ func addKnownTypes() {
 		&ThirdPartyResourceList{},
 		&ThirdPartyResourceData{},
 	)
+
+	// Add common types
+	api.Scheme.AddKnownTypes("v1", &unversioned.Status{})
```

Comment:
AddKnownTypeWithName will not help, s.typeToVersion is over written if you register the same type to different versions. https://github.com/kubernetes/kubernetes/blob/master/pkg/conversion/scheme.go#L141

Anyway, it may just be fine, I don't see how it can cause any real problem.